config/initializers/branded_organization.rb

```diff
@@ -0,0 +1,2 @@
+DEFAULT_BRANDED_ORG_ID = 246
+Rails.application.config.branded_organization_id = Rails.env.test? ? -1 : (Redis.current.get('branded_organization_id') || DEFAULT_BRANDED_ORG_ID)
```

Comment:
Could you please add some documentation for that `-1` magic number?

Can't we just:
```ruby
unless Rails.env.test?
  Rails.application.config.branded_organization_id = (Redis.current.get('branded_organization_id') || DEFAULT_BRANDED_ORG_ID)
end
```